exampleClientSmithy/smithy/example.smithy

```diff
@@ -0,0 +1,18 @@
+$version: "2.0"
+
+namespace example
+
+use jsonrpclib#jsonRPC
+use lsp#InitializeOp
+use lsp#Initialized
+use lsp#TextDocumentDidOpen
+
+@jsonRPC
+service ExampleLspClient {
```

Comment:
This right here is why I think jsonrpclib should let you implement a partial subset of endpoints, like smithy4s-bsp originally allowed you to do

Reply:
gotta make a builder based on the operation types rather than service impls, it'd be used in a way similar to langoustine endpoints. No rush with it tho, I think